tests/phpunit/MediaWiki/HooksTest.php

```diff
@@ -1389,7 +1389,7 @@ public function callSoftwareInfo( $instance ) {
 	public function callBlockIpComplete( $instance ) {
 		$handler = 'BlockIpComplete';
 
-		$block = $this->getMockBuilder( '\Block' )
+		$block = $this->getMockBuilder( '\MediaWiki\Block\Block' )
```

Comment:
This is better:

`$block = $this->createMock( Block::class );`

(with an import statement at the top of file)

Reply:
Thanks, done.